input/fsh/bw-lab-Terminology.fsh

```diff
@@ -33,3 +33,43 @@ Description:  "The status of the diagnostic report, subset from https://www.hl7.
 * DS#diagnostic-report-status-unknown "Unknown"
 * DS#diagnostic-report-status-canceled "Canceled"
 * DS#diagnostic-report-status-amended "Amended"
+
+
+
+/*
```

Comment:
@tkgakge here's an example mapping of a PIMS lookup table. You can add other mappings in this file.